caddy/caddy.go

```diff
@@ -77,6 +79,16 @@ func parse(c *caddy.Controller) (txtdirect.Config, error) {
 			}
 			resolver = resolverAddr[0]
 
+		case "proxy":
+			toProxy := c.RemainingArgs()
+			if len(toProxy) != 1 {
+				return txtdirect.Config{}, c.ArgErr()
+			}
+			upstreams, err := proxy.NewStaticUpstreams(c.Dispenser, toProxy[0])
```

Comment:
c.Dispenser will error out here as it doesn't get an actual proxy configuration. Let's remove the config logic first and use something hardcoded:
```
upstreams, err := proxy.NewStaticUpstreams(caddyfile.NewDispenser("Testfile", strings.NewReader(`
			proxy / example.com
		`)), "")
```

What that basically does:
* don't set a host (simplification)
* create a new dispenser with the most simple configuration

Reply:
I added a hardcoded example to start. Here is my local config that I used to test it:

In the txt record:
```
_redirect.two.one     60 IN TXT     "v=txtv0;to=http://google.com;type=proxy;code=302"
```

In the caddyfile:
```
example.test:8080 {
  tls off
  txtdirect {
    enable path host proxy
    resolver 127.0.0.1:5353
    proxy / google.com
    logfile stdout
  }
  log / stdout "{remote} - [{when_iso}] \"{method} {uri} {proto}\" {status} {size} {latency}"
  errors stdout
}
```